src/client/components/pages/Reservations/Reservations.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useEffect } from "react";
+import { getReservations } from "../../../data/reservations";
+import "./Reservations.css"
+
+const Reservations = (props) => {
+    const [reservations, setReservations] = useState([]);
+    const [inputState, setInputstate] = useState(false);
+    const [name, setName] = useState("");
+    const [phoneNumber, setPhoneNumber] = useState("");
+    const [numberOfGuests, setNumberOfGuests] = useState("");
+    const [email, setEmail] = useState("");
+    const [mealId, setMealId] = useState("");
+    const [date, setDate] = useState("");
```

Comment:
You can set this data up like you did for adding meals.
So as an inputValues object.

Reply:
yes, I tried that and I had some problems in returning the data but I will take more time now to change it accordingly